File: src/_score/score_module.php
<?php
namespace Materia;

class Score_Modules_PrivilegeWalk extends Score_Module
{
	public $allow_distribution = true;

	public function check_answer($log)
	{
		$answers = $this->questions[$log->item_id]->answers;
		foreach($answers as $answer)
		{
			if ($log->text == $answer['text'])
			{
				return $answer['value'];
			}
		}
		return 0;
	}

}

File: src/_score/score_module.py
from scoring.module import ScoreModule

class PrivilegeWalk(ScoreModule):
    def check_answer(self, log):
        answers = self.get_question_by_item_id(log.item_id)["answers"]
        for answer in answers:
            if log.text == answer["text"]:
                return answer["value"]

        return 0

    def calculate_score(self):
        self.calculated_percent = 100
